cli/src/semgrep/app/scans.py

```diff
@@ -290,6 +331,16 @@ def report_findings(
             f"{state.env.semgrep_url}/api/agent/scans/{self.scan_id}/findings_and_ignores",
             json=findings_and_ignores,
         )
+        # TODO: delete this once all on-prem app instances are gone
+        if response.status_code == 404:
+            response = state.app_session.post(
+                f"{state.env.semgrep_url}/api/agent/scans/{self.scan_id}/ignores",
+                json={"findings": ignores},
+            )
+            response = state.app_session.post(
+                f"{state.env.semgrep_url}/api/agent/scans/{self.scan_id}/findings",
+                json=findings_and_ignores,
```

Comment:
This should just be findings right?

Reply:
Yeah, but technically it's the same, just minus the "ignores" key so I figured we could just re-use this (sending extra fields doesn't hurt functionality)

Reply:
Ah I see